### ISSUE TITLE
Tweak Installer causes the issue after installing tweak

### ISSUE THREAD

**Issue by prafulbusa** [josephwalden13/tweak-installer#14]
And then tweak installer can't fix the issue by removing (uninstall)ing the deb from the iDevice. Any idea, how to fix the stuck in Safe mode problem? I don't wanna start over every time when I have such issue. I remember that on Cydia, it was easy to remove the unwanted tweak but after using this tool, it's not that easy. So, how do you manually remove those unwanted files (installed by this tool)?
Thank you for trying...

**Comment by josephwalden13** [comment]
You can easily uninstall it using my tool. Which error does it give when uninstalling? I’ll help to get it working again. To manually remove each file would be hard because some tweaks install files all over the filesystem

**Comment by prafulbusa** [comment]
I tried to remove (uninstall) using your tool but it didn't work either. There is no error popping on the computer. It's just iPhone SB (Springboard) stays in Safe Mode. I tried to find the tweak file/s inside the library folder (MobileSubstrate, PreferenceBundle, PreferenceLoader) but it's not anywhere in there. However, I looked other folders too but there is no clue (leftover files) found of this tweak). btw, I was trying to install [byebyehud_1.1.3.deb], I didn't do uncompress or anything. so, I did wrong? what should I do? I can start over but I am trying to find a solution first that might help others before I decide to work from the clean installation of Electra JB.  Big Thanks to you Joseph.

**Comment by josephwalden13** [comment]
[script.txt](https://github.com/josephwalden13/tweak-installer/files/1665744/script.txt)
Download this and move it to your device using WinSCP. Then run "sh /path/to/script.txt" via ssh. It'll fully remove the tweak for you. My tool should have done this automatically, but if it failed this'll work. If this doesn't work it must be another tweak causing it. I'd try to uninstall other recent tweaks if thats the case

**Comment by prafulbusa** [comment]
I am ready to move but what location I should execute this? inside the library?

**Comment by josephwalden13** [comment]
You can execute it wherever you like, as long as the command contains the correct path

**Comment by prafulbusa** [comment]
I tried to execute that file, also converted to .sh and ran but it didn't work because it didn't find any leftover files from these locations, so I manualy tried to remove them but I didn't find anything. 
rm /bootstrap/Library/PreferenceBundles/ByeByeHUD.bundle/BBH.png
rm /bootstrap/Library/PreferenceBundles/ByeByeHUD.bundle/BBH\@2x.png
rm /bootstrap/Library/PreferenceBundles/ByeByeHUD.bundle/BBHLogo.png
rm /bootstrap/Library/PreferenceBundles/ByeByeHUD.bundle/ByeByeHUD
rm /bootstrap/Library/PreferenceBundles/ByeByeHUD.bundle/clearPic.png
rm /bootstrap/Library/PreferenceBundles/ByeByeHUD.bundle/high.png
rm /bootstrap/Library/PreferenceBundles/ByeByeHUD.bundle/high\@2x.png
rm /bootstrap/Library/PreferenceBundles/ByeByeHUD.bundle/Info.plist
rm /bootstrap/Library/PreferenceBundles/ByeByeHUD.bundle/low.png
rm /bootstrap/Library/PreferenceBundles/ByeByeHUD.bundle/low\@2x.png
rm /bootstrap/Library/PreferenceBundles/ByeByeHUD.bundle/mute.png
rm /bootstrap/Library/PreferenceBundles/ByeByeHUD.bundle/mute\@2x.png
rm /bootstrap/Library/PreferenceBundles/ByeByeHUD.bundle/ring.png
rm /bootstrap/Library/PreferenceBundles/ByeByeHUD.bundle/ring\@2x.png
rm /bootstrap/Library/PreferenceBundles/ByeByeHUD.bundle/Root.plist
rm /bootstrap/Library/PreferenceLoader/Preferences/ByeByeHUD.plist
rm /usr/lib/SBInject/ByeByeHUD.dylib
rm /usr/lib/SBInject/ByeByeHUD.plist

**Comment by josephwalden13** [comment]
That is every file the tweak installs, so it was most likely another tweak. Do you keep the deb files after installing them? If so you could remove them to see which one causes it, however if not run 'ls /bootstrap/Library/SBInject' and send me the output and I'll see if I can find out which tweak is causing it :)

**Comment by prafulbusa** [comment]
![image](https://user-images.githubusercontent.com/35320049/35414682-028bcdfe-01d8-11e8-93f3-c68d7046b44a.png)

**Comment by prafulbusa** [comment]
now, I am looking each directly by this search

**Comment by josephwalden13** [comment]
It'd be easier to just ls /bootstrap/Library/SBInject and see if any of those are incompatible. ColorFlow3 by any chance?

**Comment by prafulbusa** [comment]
I keep that .deb file on my computer, but I didn't put it on my iPhone. as I was using tweak installer tool, I browsed from my computer and then executed files on iPhone.  That's all I know. However, that picture I send you is still searching (each and every folder) on iDevice. hopefully, we will have something in the result. I have output of 'ls' on 
![image](https://user-images.githubusercontent.com/35320049/35415029-1b399d62-01d9-11e8-81db-bf7b74308c8d.png)

**Comment by prafulbusa** [comment]
only two tweaks I see here, InApp(free in-app purchase), Threedeeboard (barrel) and both of them were working fine.

**Comment by josephwalden13** [comment]
Hmm. It must be one of those. My guess is InApp. Try removing it using the deb.

**Comment by prafulbusa** [comment]
Cool, I will remove one by one and see what causes the issues. I will start with InApp, and then barrel, I will update once I am done playing with these files. Thanks for being here to help

**Comment by josephwalden13** [comment]
Ok, no problem :) Happy to help

**Comment by josephwalden13** [comment]
Did you get it working in the end?

**Comment by prafulbusa** [comment]
Yes, I believe that might be because of Barrel tweak. After removing both tweak( coz I had only two at that time, Barrel and InApp) which gave me chance to start over and I started installing other tweaks and was Barrel whic again started behaving like before so I removed it from the iPhone and instead I just installed Cylinder tweak which is alternate of Barrel. However, I didn’t use the tool to install tweak at this time but WinScp to transfer Library files in necessary folders like preferences loader, pref bundles and in SBinject. Now I have more than 20 tweaks, and learned how to handle them. It was easy and painless. By the way, That search for ByeByeHuD using WinScp was on going and I had to stop few time and that’s why I didn’t let it finish. Thank you for your help and let me know if I can be any help.
